components.yaml

```
@@ -5,54 +5,54 @@ GEOSadas:
 env:
   local: ./@env
   remote: ../ESMA_env.git
-  tag: v1.4.3
+  tag: v3.2.1
   develop: main
 
 cmake:
   local: ./@cmake
   remote: ../ESMA_cmake.git
-  tag: v1.0.11
+  tag: v3.4.2
   develop: develop
 
 ecbuild:
   local: ./@cmake/@ecbuild
   remote: ../ecbuild.git
-  tag: geos/v1.0.0
+  tag: geos/v1.0.6
 
 NCEP_Shared:
   local: ./src/Shared/@NCEP_Shared
   remote: ../NCEP_Shared.git
-  tag: v1.1.0
+  tag: rt-GEOSadas-5_29_0_prep3
   develop: main
 
 GMAO_Shared:
   local: ./src/Shared/@GMAO_Shared
   remote: ../GMAO_Shared.git
-  tag: cvs/GEOSadas-5_27_1_p3
+  tag: rt-GEOSadas-5_29_0_prep3
   develop: dev/adas
 
 MAPL:
   local: ./src/Shared/@MAPL
   remote: ../MAPL.git
-  tag: v1.1.14
+  tag: v2.6.7
```

```suggestion
  tag: v2.7.0
```

This needs to be v2.7.0 because GEOSgcm_App requires and interface change. The differences between v2.6.7 and v2.7.0 are this interface change and some bugfixes, including a bug in MAPL_Shmem

https://github.com/GEOS-ESM/MAPL/compare/v2.6.7..v2.7.0